src/index.js

```diff
@@ -0,0 +1,36 @@
+import header from './components/header';
+import main from './components/main';
+import menu from './components/menu';
+import contact from './components/contact';
+
+
+header();
+main();
+menu();
+contact();
+
+const mainButton = document.getElementById('home-button');
+const menuButton = document.getElementById('menu-button');
+const contactButton = document.getElementById('contact-button');
+
+const mainPage = document.getElementById('main-page');
+const menuPage = document.getElementById('menu-page');
+const contactPage = document.getElementById('contact-page');
+
+mainButton.addEventListener('click', () => {
+  mainPage.style.display = 'block';
+  menuPage.style.display = 'none';
+  contactPage.style.display = 'none';
```

Comment:
Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)